im.nheko.Nheko.yaml

```diff
@@ -0,0 +1,247 @@
+id: im.nheko.Nheko
+command: im.nheko.Nheko
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+finish-args:
+  - --device=dri
+  # needed for webcams, see #517
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it's either wayland with fallback or x11 alone.

Reply:
Would the package then still work on systems without xwayland?

Reply:
that's what the fallback is about.

Reply:
`wayland` is NOT for `Xwayland` either.

Reply:
We currently want to run on x11 by default, because we need the x11 socket to share x11 windows. We can't use fallback-x11 in that case. However, if a system is configured without Xwayland and only supporting wayland apps, specifying the wayland socket should still allow Nheko to launch, only dropping the support for sharing the non-existing x11 windows.

So as far as I am concerned, "x11, wayland" is the correct socket configuration, since both work, but we need the x11 socket for screensharing. Am I misunderstanding something?

Reply:
no. the correct setting is `x11` and no `wayland`. A system without Xwayland broken.

Reply:
Nheko has quite a few users without xwayland, I would prefer to not break it for no reason?

Reply:
then you should enable wayland for everyone. it either work or doesn't

Reply:
> since both work, 

Unless you set QT_QPA_PLATFORM=wayland, Qt with this configuration will use Xwayland by default in a wayland session.

But since you can't set that, using x11 only seems fine? Those who have no Xwayland, will need to run with `--env=QT_QPA_PLATFORM=wayland` (or as an override) anyways. They can add `--socket=wayland` there as well?

Reply:
On my systems Qt seems to fallback to wayland transparently, but I can remove it and see, if people start complaining.

Reply:
as I said, if it works on wayland the proper permissions are `wayland + fallback-x11`

Reply:
No, they are not, because that breaks screensharing X11 windows, since you need the x11 socket for that. They might work using pipewire, but we haven't tested that enough yet.

Reply:
> On my systems Qt seems to fallback to wayland transparently,

Yea seems to be the case for me too. I guess they didn't update the wiki.

Reply:
Changed it https://github.com/flathub/flathub/pull/5328/commits/ea17b558f75d69a852ed7afa5e364c0f80ab6ecb